src/setup/i18n.ts

```diff
@@ -8,13 +8,22 @@ export const I18N_FALLBACK = 'de';
 export const I18N_FALLBACK_MESSAGES = fallbackMessages;
 export const I18N_LOCALES = [I18N_FALLBACK, 'fr'];
 
+const datetimeFormats: IntlDateTimeFormat = {
+  month: {
```

Comment:
todo: please also add a hint comment here. I guess it is `// January, February, ...`